### DISCUSSION TITLE
Open big files in TIFF format with a single strip

### DISCUSSION THREAD

**Discussion by DavidStorm** [libvips/libvips#2322] [General]
We have a TIFF file of 2,97 GB made in a single strip and current TIFFClientOpen mode options can't read this image. In file tiff.c line 215 there is the open call as:
	if( !(tiff = TIFFClientOpen( "source input", "rm",
to open this kind of files, just add 'C' option:
	if( !(tiff = TIFFClientOpen( "source input", "rmC",
to enable the use of ''strip chopping''.

**Comment by jcupitt** [comment]
Hi @DavidStorm, that's interesting, I hadn't come across the strip chopping flag. I've made this change in 8.11.1 and credited you. Thanks!

**Reply by DavidStorm** [reply]
Thanks to you @jcupitt
